recipe/meta.yaml

```
@@ -1,36 +1,38 @@
-{% set version = "23.6.0" %}
+{% set version = "32.0.1" %}
 
 package:
   name: python-kubernetes
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/k/kubernetes/kubernetes-{{ version }}.tar.gz
-  sha256: cadb1cd7c44eae5b39b50316313343c0f2aff94529f067ec2ba44d38411158e3
+  url: https://pypi.org/packages/source/k/kubernetes/kubernetes-{{ version }}.tar.gz
+  sha256: 42f43d49abd437ada79a79a16bd48a604d3471a117a8347e87db693f2ba0ba28
 
 build:
   number: 0
-  script: {{ PYTHON }} -m pip install . -vv
-  skip: True  # [py<36]
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation --ignore-installed --no-cache-dir -vv
+  skip: True  # [py<38]
 
 requirements:
   host:
     - python
     - pip
-    - setuptools
+    # - setuptools
     - wheel
   run:
     - python
     - certifi >=14.05.14
-    - google-auth >=1.0.1
+    - six >=1.9.0
     - python-dateutil >=2.5.3
+    - setuptools >=21.0.0
     - pyyaml >=5.4.1
+    - google-auth >=1.0.1
+    - websocket-client >=0.32.0, !=0.40.0, !=0.41.*, !=0.42.*
     - requests
     - requests-oauthlib
-    - setuptools >=21.0.0
-    - six >=1.9.0
+    - oauthlib >=3.2.2
     - urllib3 >=1.24.2
-    - websocket-client >=0.32.0,!=0.40.0,!=0.41.*,!=0.42.*
+    - durationpy >=0.7
```

```suggestion
    - durationpy >=0.7
  run_constrained:
    - adal >=1.0.2
```
https://github.com/kubernetes-client/python/blob/v32.0.1/setup.py#L30-L32